src/Processors/Formats/Impl/Parquet/Write.h

```diff
@@ -84,6 +85,7 @@ struct ColumnChunkWriteState
     int compression_level = 3;
     Int64 datetime_multiplier = 1; // for converting e.g. seconds to milliseconds
     bool is_bool = false; // bool vs UInt8 have the same column type but are encoded differently
+    DataTypePtr enum_type = nullptr;
```

Comment:
There is a field `type` above, no need for a new field for enum type

Reply:
For some reason I missed that field, thanks for the suggestion :+1: